### PR TITLE
systemd-inhibit: add page

### PR DESCRIPTION
**Pull request by rothgar**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
253

Adds command to #10191

### CODE REVIEW

File: pages/linux/systemd-inhibit.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+> Prohibit system from entering certain power states.
+> Inhibitor locks may be used to block or delay system sleep and shutdown requests as well as automatic idle handling.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-inhibit.html>.
+
+- List all active inhibition locks instead of acquiring one:
+
+`systemd-inhibit --list`
+
+- Block system shutdown with sleep command:
```
Comment (gutjuri) [inline pages/linux/systemd-inhibit.md:11]:
```suggestion
- Block system shutdown for {{5}} seconds with sleep command:
```

**Review by gutjuri — CHANGES_REQUESTED** [review]

**Review by gutjuri — APPROVED** [review]
Thanks!

File: pages/linux/systemd-inhibit.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+> Prohibit system from entering certain power states.
+> Inhibitor locks may be used to block or delay system sleep and shutdown requests as well as automatic idle handling.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-inhibit.html>.
+
+- List all active inhibition locks instead of acquiring one:
+
+`systemd-inhibit --list`
+
+- Block system shutdown for {{5}} seconds with sleep command:
```
Comment (kbdharun) [inline pages/linux/systemd-inhibit.md:11]:
```suggestion
- Block system shutdown for a specified number of seconds with the `sleep` command:
```

File: pages/linux/systemd-inhibit.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+`systemd-inhibit --who $USER --what shutdown sleep {{5}}`
+
+- Keep system from sleeping or idling until after download is finished:
```
Comment (kbdharun) [inline pages/linux/systemd-inhibit.md:15]:
```suggestion
- Keep the system from sleeping or idling until after the download is completed:
```

File: pages/linux/systemd-inhibit.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# systemd-inhibit
+
+> Prohibit system from entering certain power states.
```
Comment (kbdharun) [inline pages/linux/systemd-inhibit.md:3]:
```suggestion
> Prohibit the system from entering certain power states.
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

**Review by gutjuri — APPROVED** [review]

File: pages/linux/systemd-inhibit.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# systemd-inhibit
+
+> Prohibit the system from entering certain power states.
+> Inhibitor locks may be used to block or delay system sleep and shutdown requests as well as automatic idle handling.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-inhibit.html>.
+
+- List all active inhibition locks instead of acquiring one:
```
Comment (acuteenvy) [inline pages/linux/systemd-inhibit.md:7]:
```suggestion
- List all active inhibition locks:
```

File: pages/linux/systemd-inhibit.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+`systemd-inhibit --list`
+
+- Block system shutdown for a specified number of seconds with the `sleep` command:
+
+`systemd-inhibit --who $USER --what shutdown sleep {{5}}`
+
+- Keep the system from sleeping or idling until after the download is completed:
```
Comment (acuteenvy) [inline pages/linux/systemd-inhibit.md:15]:
```suggestion
- Keep the system from sleeping or idling until the download is complete:
```

File: pages/linux/systemd-inhibit.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+`systemd-inhibit --what sleep:idle wget {{https://example.com/file}}`
+
+- Ignore lid close switch until script exits:
```
Comment (acuteenvy) [inline pages/linux/systemd-inhibit.md:19]:
```suggestion
- Ignore lid close switch until the script exits:
```

File: pages/linux/systemd-inhibit.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+> Inhibitor locks may be used to block or delay system sleep and shutdown requests as well as automatic idle handling.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-inhibit.html>.
+
+- List all active inhibition locks instead of acquiring one:
+
+`systemd-inhibit --list`
+
+- Block system shutdown for a specified number of seconds with the `sleep` command:
+
+`systemd-inhibit --who $USER --what shutdown sleep {{5}}`
```
Comment (acuteenvy) [inline pages/linux/systemd-inhibit.md:13]: Moved to another example to describe the option.
```suggestion
`systemd-inhibit --what shutdown sleep {{5}}`
```

File: pages/linux/systemd-inhibit.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+
+`systemd-inhibit --what sleep:idle wget {{https://example.com/file}}`
+
+- Ignore lid close switch until script exits:
+
+`systemd-inhibit --what sleep:handle-lid-switch {{path/to/script}}`
+
+- Ignore power button press while command is running:
+
+`systemd-inhibit --what handle-power-key {{command}}`
```
Comment (acuteenvy) [inline pages/linux/systemd-inhibit.md:25]:
```suggestion
`systemd-inhibit --what handle-power-key {{command}}`

- Describe who and why created the inhibitor (default: the command and its arguments for `--who` and `Unknown reason` for `--why`):

`systemd-inhibit --who {{$USER}} --why {{reason}} --what {{operation}} {{command}}`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**